lib/signaling/v2/peerconnection.js

```diff
@@ -830,7 +832,7 @@ class PeerConnectionV2 extends StateMachine {
         // support, we have to use the same hacky solution as Safari. Revisit
         // this when RTCRtpTransceivers and MIDs land. We should be able to use
         // the same technique as Firefox.
-        : isSafari || this._isUnifiedPlan ? new OrderedTrackMatcher() : new IdentityTrackMatcher();
+        : isSafari || isiOSChrome || this._isUnifiedPlan ? new OrderedTrackMatcher() : new IdentityTrackMatcher();
```

Comment:
@PikaJoyce `isUnifiedPlan` will cover the iOS chrome case, so we don't need the extra check.
```suggestion
        : isSafari || this._isUnifiedPlan ? new OrderedTrackMatcher() : new IdentityTrackMatcher();
```